src/transport.ts

```diff
@@ -84,8 +84,15 @@ export class PinoSentryTransport {
   sentryExceptionLevels = [Severity.Fatal, Severity.Error];
   decorateScope = (_data: Record<string, unknown>, _scope: Sentry.Scope) => {/**/};
 
-  public constructor(options?: PinoSentryOptions) {
-    Sentry.init(this.validateOptions(options || {}));
+  
+  public constructor(options?: PinoSentryOptions & Sentry.NodeOptions)
+  public constructor(options?: PinoSentryOptions, initializeSentry: false)
+  public constructor(options?: any, initializeSentry?: boolean = true) {
+    const validatedOptions = this.validateOptions(options || {});
+
+    if (initializeSentry) {
```

Comment:
what about having this set via `options`? also the what would `get sentry` return below?

the only way oi see this to work, if you pass an instance of Sentry to the constructor. and use that value in rest of the code.